gnomad/utils/constraint.py

```diff
@@ -577,6 +577,8 @@ def build_models(
     high_cov_definition: int = COVERAGE_CUTOFF,
     upper_cov_cutoff: Optional[int] = None,
     skip_coverage_model: bool = False,
+    coverage_metric: str = "exomes_AN_percent",
```

Comment:
Let's make `exome_coverage` the default, or better yet, change to take an expression

Reply:
would an expression work here when it's being referenced in multiple different hts?